packages/sources/eth-balance/test/e2e/balance.test.ts

```diff
@@ -0,0 +1,91 @@
+import { assertSuccess } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+import { serverErrors, validationErrors } from '../../../../core/test-helpers/src/helpers'
+
+describe('execute', () => {
```

Comment:
Can we also add unit and integration tests (that uses nock to mock the RPC endpoint)?